packages/metrics/src/features/audit/handler.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import * as Hapi from '@hapi/hapi'
+import { generateAuditPoint } from '@metrics/features/registration/pointGenerator'
+import { writePoints } from '@metrics/influxdb/client'
+import { internal } from '@hapi/boom'
+import { IUserAuditBody } from '@metrics/features/registration'
+import { PRACTITIONER_ID } from '@metrics/features/getTimeLogged/constants'
+import { getUserAuditEvents } from './service'
+
+export async function newAuditHandler(
+  request: Hapi.Request,
+  h: Hapi.ResponseToolkit
+) {
+  const points = []
+  try {
+    const remoteAddress =
+      request.headers['x-real-ip'] || request.info.remoteAddress
+    const userAgent =
+      request.headers['x-real-user-agent'] || request.headers['user-agent']
+    const payload = request.payload as IUserAuditBody
+    points.push(
+      generateAuditPoint(
+        payload.practitionerId,
+        payload.action,
+        remoteAddress,
+        userAgent,
+        payload.additionalData
+      )
+    )
+    await writePoints(points)
+  } catch (err) {
+    return internal(err)
+  }
+  return h.response().code(201)
+}
+
+export async function getUserAuditsHandler(request: Hapi.Request) {
+  const practitionerId = request.query[PRACTITIONER_ID]
+  const skip = request.query['skip'] || 0
+  const size = request.query['count']
+
+  const results = await getUserAuditEvents(practitionerId, size, skip)
+  return {
+    results,
+    total: results.length
```

Comment:
@KaliGreen notice that this actually needs to return the full total number for events for this practitioner

See the previous implementation:
https://github.com/opencrvs/opencrvs-core/blob/6942413c4de9a429dd1686b8dd4705834a694a20/packages/metrics/src/features/getTimeLogged/handler.ts#L46-L58

Reply:
i will push that up in my next commit